client/landing/gutenboarding/onboarding-block/style-preview/style.scss

```diff
@@ -35,12 +35,13 @@
 
 .style-preview__font-option {
 	min-height: 3.4em;
-	box-shadow: inset 0 0 0 1px var( --studio-gray-10 );
 	justify-content: center;
-
+	
 	// Extra specificity to override core style
 	// This is effectively a more-specific synonmy of the same selector
 	&.components-button {
+		color: var( --studio-gray-90 );
+		box-shadow: inset 0 0 0 1px var( --studio-gray-10 );
```

Comment:
This appears to be the current styling, I don't believe it results in any change.